src/injected/visualization/accessible-names-formatter.ts

```diff
@@ -40,11 +40,9 @@ export class AccessibleNamesFormatter implements Formatter {
                 text: formattedAccessibleName,
                 fontWeight: '400',
                 fontSize: '10px',
-                outline: '3px dashed',
```

Comment:
this might impact the style, please retest with and without these changes

Reply:
outline property was removed from its interface DrawerConfiguration in its previous commits. Tried checking in UI but could not find the relevant page for the property.

Reply:
With respect to this PR this property is removed, found about it from checking DrawerConfiguration interface git history
https://github.com/microsoft/accessibility-insights-web/pull/6164/files#diff-4f7770a7abf700b18af7cef9727d4b762288cd84ba55f39161a89d9f86d99cd9